wrapper/src/main/java/software/amazon/jdbc/dialect/DialectManager.java

```diff
@@ -44,60 +47,124 @@ public class DialectManager implements DialectProvider {
           put("mysql", new MysqlDialect());
           put("pg", new PgDialect());
           put("mariadb", new MariaDbDialect());
+          put("rds-mysql", new RdsMysqlDialect());
+          put("rds-pg", new RdsPgDialect());
           put("aurora-mysql", new AuroraMysqlDialect());
           put("aurora-pg", new AuroraPgDialect());
         }
```

Comment:
no need for the second braces

Reply:
It seems that double braces is the only option for Java 8. https://www.baeldung.com/java-initialize-hashmap